### DISCUSSION TITLE
`huh.Text`, ^e, open in `$EDITOR`, and Emacs keybindings

### DISCUSSION THREAD

**Discussion by meowgorithm** [charmbracelet/huh#590] [Ideas]
So typically in most command line input situations <kbd>ctrl+e</kbd> goes to the end of the line (by way of being a standard emacs keybinding). This is even true in input fields on macOS.

Given that, <kbd>ctrl+e</kbd> shouldn't open the buffer in the `$EDITOR` by default and we should choose another keybinding. Nothing comes to mind off the cuff, but note that by default <kbd>alt</kbd> isn’t available in some terminals, particularly on macOS.

**Comment by radar07** [comment]
Makes sense! How about `ctrl+o` or `ctrl+i` to open the EDITOR?

**Comment by bashbunni** [comment]
I'm going to convert this to a discussion, once we're ready to work on it we can bring it back to our backlog of issues :) Please feel free to leave your thoughts here. We will be looking at them when we're ready to adopt new features in this project!
